perl/lib/Seq/Output.pm

```diff
@@ -131,13 +161,13 @@ sub makeOutputString {
               ref $_
               ?
                 # at this position this feature has multiple values
-                join( $valDelim, map { defined $_ ? $_ : $missChar } @$_ )
+                join( $valDelim, map { defined $_ ? $_ : $missChar } @{ uniqueify($_) } )
```

Comment:
Maybe I just don't know perl, but I got tripped up on this line.  It looks like $_ is being used in two different scopes here? (the scope of $_ passed to uniqueify and an inner scope for the map).  Pls disregard if this is an idiomatic construct in perl but I would consider recasting this line to avoid the two-scopes problem if possible.